.github/workflows/build_hub.yml

```diff
@@ -103,10 +88,26 @@ jobs:
 
       - name: Spin up containers
         run: |
-          docker-compose -f docker-compose.yml up -d prime_dev redox ftps sftp azurite
+          docker-compose -f docker-compose.yml up -d prime_dev redox ftps sftp azurite settings
```

Comment:
Context: gradle reloadSettings does not have a wait function yet, so we must use the settings container to make sure we do not get a failure.  See #3271